app/helpers/alexa_interface_helper.rb

```diff
@@ -16,6 +16,52 @@ def create_response(call_parameters)
     response_for_alexa.build_response
   end
 
+  # this method and the one above could probably be re factored to be more similar or even joined into one method maybe? (search time is a concern...)
+  def category_search_response
+    given_category = params["request"]["intent"]["slots"]["category"]["value"]
+    category = @lookup_hash[given_category] # I think this is right?
+    if category
+      response_for_alexa = AlexaRubykit::Response.new
+      response = category_call({location: get_location[:location], category: category})
+      not_started = select_not_started(response)
+      top_ten = pick10(not_started)
+      top_one = pick1(top_ten)
+      format_category_speech_for_alexa(response_for_alexa, top_one, given_category)
+      format_text_for_alexa(response_for_alexa, top_ten)
+      response_for_alexa.build_response
+    else
+      generate_bad_category_response(given_category)
+    end
+  end
+
+  def category_call(call_parameters = {})
+    # page size is 10 for testing; should be ~500 for production
+    if Rails.env.production?
+      page_size = "50"
+    else
+      page_size = "10"
+    end
+
+    parameters_hash = {category: call_parameters[:category], location: call_parameters[:location], date: "Today", sort_order: "popularity", mature: "normal", page_size: page_size, change_multi_day_start: "true" }
+    client = EventfulApi::Client.new({})
+    response = client.get('/events/search', parameters_hash)
+    # hash > "events" > "event" > array of events
+    response["events"]["event"]
+  end
+
+  def generate_bad_category_response(category)
+    response_for_alexa = AlexaRubykit::Response.new
+    response_for_alexa.add_speech("Sorry, #{category} is not a valid category")
```

Comment:
direct them to help: categories list too, adding as a small trello card (does not have to block merge)